README.md

```diff
@@ -64,39 +62,6 @@ Requirements:
 $ mvn clean package
 ```
 
-### To deploy
```

Comment:
Yeah I was thinking that, but it currently has some annoying stuff like falling over on Sumologic if it isn't there.

That's why I initially had the lambda directory + symlinks sitting there, to attempt to smooth over deployment *without* our internal processes.

Reply:
gah! That's annoying! Anyone running it up in dev won't even want the sumologic stuff, but I suppose not having the code there will still break the package command :(

Reply:
Yeah the template isn't even going to use it, but package complains because I don't think it pays attention to the fact that it's conditional. 

Could have a dummy `lambda/sumologic` directory, but it still depends on having compiled versions of the Lambdas, so the doco can really start growing organically to describe a deployment process we don't even use...